docs/advanced-config.md

```diff
@@ -39,6 +39,8 @@ export SPLUNK_METRICS_ENDPOINT=https://ingest.us0.signalfx.com
 | `otel.exporter.jaeger.endpoint` | `OTEL_EXPORTER_JAEGER_ENDPOINT` | `http://localhost:9080/v1/trace` | Stable  | The Jaeger endpoint to connect to. Setting this will override the `splunk.realm` property.                                               |
 | `otel.traces.exporter`          | `OTEL_TRACES_EXPORTER`          | `otlp`                           | Stable  | Select the traces exporter to use. We recommend using either the OTLP exporter (`otlp`) or the Jaeger exporter (`jaeger-thrift-splunk`). |
 
+:warning: **Support for the `jaeger-thrift-splunk` exporter will be removed after December 17th, 2022. See the [deprecation notice](https://github.com/signalfx/signalfx-agent/blob/main/docs/smartagent-deprecation-notice.md) on the SmartAgent for details. ** :warning:
```

Comment:
I don't think this is going to be that easy. Isn't this the recommended way for direct ingest also? Before deprecating it shouldn't there be an alternative for it? I don't think that any of the other exporters do anything with `splunk.access.token`

Reply:
I was wrong, `splunk.access.token` is also used for `otlp` exporter, so deprecating should be ok